lib/pages/people-page.js

```diff
@@ -14,6 +14,7 @@ export default class PeoplePage extends BaseContainer {
 		this.searchResultsLoadingSelector = By.css( '.people-profile.is-placeholder' );
 		this.peopleListItemSelector = By.css( '.people-list-item' );
 		this.successNoticeSelector = By.css( '.people-notices__notice.notice.is-success' );
+		this.pendingInviteSelector = By.css('.section-header__label-text');
```

Comment:
This is also the class on the "Accepted" label on the People > Invites tab, and I see it for example on the People > Team tab, so it isn't specific to the Invites tab either. Is there another label you can use that's specific to the pending invites section of the page?